src/components/Reports/TasksTable/TasksTable.jsx

```diff
@@ -22,6 +22,11 @@ export const TasksTable = ({ WbsTasksID }) => {
     users: '',
   });
 
+
+  useEffect(() => {
```

Comment:
Remove unnecessary useEffect.

Just as a tip, this useEffect is unnecessary even when testing because`isAssigned` is set in state. Whenever `isAssigned` changes, the component will rerender and all the code inside the file will rerun, including the `console.log(isAssigned)`

Reply:
Good catch! I have removed it. Thank You